packages/babel-jest/src/index.js

```diff
@@ -18,75 +18,34 @@ import type {
 import crypto from 'crypto';
 import fs from 'fs';
 import path from 'path';
+import {
+  transform as babelTransform,
+  util as babelUtil,
```

Comment:
These utils don't exist in 7.x FYI.

Reply:
@SimenB Remove the util functionality completely? I noticed the `canCompile` function being set in a few places.

Reply:
I suppose? It was probably an optimization, but I guess it doesn't matter as it's been removed

Reply:
All it did was check our list of suggested list of extensions, so we just exposed that instead: https://github.com/babel/babel/blob/master/packages/babel-core/src/index.js#L37 I'm not sure it's that useful for Jest anyway though, since you already limit it to `.js` files by default.